modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -215,7 +216,7 @@ export const TabsItem = createComponent('button')({
 
     return (
       <OverflowTooltip>
-        <StyledTabItem as={Element} {...props}>
+        <StyledTabItem as={Element} maxWidth={modality === 'touch' ? undefined : 280} {...props}>
```

Comment:
should the maxWidth be configurable?

Reply:
`maxWidth` can be overridden by passing `maxWidth` to the `Tabs.Item` component:

```tsx
<Tabs>
  <Tabs.List>
     <Tabs.Item maxWidth={300}>First Tab</Tabs.Item>
```

We don't have a system for defining properties based on screen size, component size, or modality. I would not worry about it without a system in place.